sdks/python/container/Dockerfile

```diff
@@ -80,6 +80,11 @@ RUN  \
     # Remove pip cache.
     rm -rf /root/.cache/pip
 
+# Required for using Beam Python SDK on ARM machines.
+RUN if [ "${TARGETARCH}" = "arm64" ] ; then \
+      apt-get install libgeos-dev ; \
```

Comment:
i would suggest to move it to line 42, note that apt temp files are cleared, and this command might require a prior call to `apt update`.

Also, did you run validatesconainerarm tests to verify this change ?

Reply:
> Also, did you run validatesconainerarm tests to verify this change ?

you could post a link to GHA run on the PR.

Reply:
overall, pr intent looks good to me.

Reply:
- Tried to put it around line 42 earlier but I don't think the if condition can be inserted between different package names after calling apt-get install.
- I can't run on this branch because it is just a fork of the apache/beam master not the original master branch.